### PR TITLE
ref(images loaded): Add design adjusts

### PR DESCRIPTION
**Pull request by priscilawebdev**

### CODE REVIEW

File: src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/debugImageDetails/candidate/features.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  const features = Object.entries(download.features).filter(([_key, value]) => value);
+
+  if (!features.length) {
+    return <NotAvailable />;
+  }
+
+  return (
+    <Wrapper>
+      {Object.entries(download.features)
+        .filter(([_key, value]) => value)
```
Comment (matejminar) [inline src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/debugImageDetails/candidate/features.tsx:34]: Is this the same as `features` variable?

File: src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/index.tsx [hunk 2]
```diff
@@ -300,6 +299,8 @@ class DebugMeta extends React.PureComponent<Props, State> {
   render() {
     const {filter, filteredImages: images, scrollbarSize} = this.state;
 
+    console.log('scrollbarSize', scrollbarSize);
```
Comment (matejminar) [inline src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/index.tsx:302]: woops

Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/index.tsx]: opss

File: src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/debugImageDetails/candidate/features.tsx (hunk 1 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/debugImageDetails/candidate/features.tsx:34]: I dont think so

Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/debugImageDetails/candidate/features.tsx]: yes

**Review by matejminar — APPROVED** [review]

**Pull request merged**